### PR TITLE
Add pre build step

### PR DESCRIPTION
**Pull request by lukaskwkw**
I really like this app but I had a lot of headache why it is not able to get build. Normally I work with NodeJS env and don't know much about go env.  When I was trying to build I got build errors but those didn't make sense and were not any helpfull to aim real problem which was this env variable not set to 1. That variable by defualt is 0 when installing Go on windows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,20 @@ This API server enables audio transcription using the OpenAI Whisper models.
 
 # Build from source
 
+Before build make sure that **CGO_ENABLED** env is set to **1**
+
+```
+set CGO_ENABLED 1
```
Comment (xzeldon) [inline README.md]: in powershell `set` will not work, use `$env:CGO_ENABLED = "1"` instead

File: README.md [hunk 2]
```diff
@@ -11,6 +11,20 @@ This API server enables audio transcription using the OpenAI Whisper models.
...
+```
+
+or preferable set it parament. Then check it via
```
Comment (xzeldon) [inline README.md]: maybe just make a note like "you can check this with this command"?

File: README.md [hunk 3]
```diff
@@ -11,6 +11,20 @@ This API server enables audio transcription using the OpenAI Whisper models.
...
+
+```
+go env
+```
+
+Also you have to have installed gcc x64 i.e. by MYSYS
```
Comment (xzeldon) [inline README.md:26]: I have GCC installed on my system, but I'm not sure if it's required. I will have to check this myself, thank you

upd: GCC is really necessary

**Review by xzeldon — CHANGES_REQUESTED** [review]
Thank you for choosing to share the problems you have encountered and also for suggesting solutions.

I would like you to consider my suggested changes. After which I will accept this PR.

Thanks again for your participation!

File: README.md (hunk 1 above)
Comment (lukaskwkw) [reply README.md]: Fixed. The set I think work with normal cmd

File: README.md (hunk 2 above)
Comment (lukaskwkw) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (lukaskwkw) [reply README.md:26]: On my clean Windows 11 Pro installation I didn't get gcc preinstalled and when I tried to build after I setting the env variable I got error that gcc command couldn't be found

**Pull request merged**